From 41fc4faf3d711279e5738d730c8d42f4886f9392 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 16 May 2024 20:17:12 +0200
Subject: [PATCH 0848/1695] dts: stm32: remove the pins-are-numbered property

Remove the deprecated property "pins-are-numbered" from pinctrl and
pinctrl_z nodes of stm32mp soc to ensure to be conform with the
upstream series of the link below.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d9826f8916a3a41c9a2f072078b96c31cbda76a8 dts: stm32: remove the pins-are-numbered property]
Link: https://patchwork.kernel.org/project/linux-arm-kernel/list/?series=69786
Change-Id: I353518149d2f9268def2fda060d709d12a951553
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/380449
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../devicetree/bindings/pinctrl/st,stm32-pinctrl.yaml          | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/documentation/devicetree/bindings/pinctrl/st,stm32-pinctrl.yaml b/documentation/devicetree/bindings/pinctrl/st,stm32-pinctrl.yaml
index 8ffb95f63..3be1efc83 100644
--- a/documentation/devicetree/bindings/pinctrl/st,stm32-pinctrl.yaml
+++ b/documentation/devicetree/bindings/pinctrl/st,stm32-pinctrl.yaml
@@ -330,7 +330,6 @@ required:
   - '#address-cells'
   - '#size-cells'
   - ranges
-  - pins-are-numbered
 
 additionalProperties: false
 
@@ -344,7 +343,6 @@ examples:
               #size-cells = <1>;
               compatible = "st,stm32f429-pinctrl";
               ranges = <0 0x40020000 0x3000>;
-              pins-are-numbered;
 
               gpioa: gpio@0 {
                       gpio-controller;
@@ -362,7 +360,6 @@ examples:
               #size-cells = <1>;
               compatible = "st,stm32f429-pinctrl";
               ranges = <0 0x50020000 0x3000>;
-              pins-are-numbered;
 
               gpiob: gpio@1000 {
                       gpio-controller;
-- 
2.47.2

